setup.py

```diff
@@ -1,4 +1,60 @@
 from setuptools import find_packages, setup
+"""
+Check list to release a new version.
+
+1. Checkout the main branch and pull the latest changes:
+   ```
+   git checkout main
+   git pull
+   ```
+
+2. Checkout a new branch with the version that you'd like to release: v<MINOR-VERSION>-release,
+   for example `v0.5-release`. All patches will be done to that same branch.
+
+3. Update the `__version__` variable in the `src/huggingface_hub/__init__.py` file to point 
+   to the version you're releasing:
+   ```
+   __version__ = "<VERSION>"
+   ```
+   
+4. Make sure that the conda build works correctly by building it locally:
+   ```
+   conda install -c defaults anaconda-client conda-build
+   HUB_VERSION=<VERSION> conda-build .github/conda
+   ```
```

Comment:
do we want to maintain conda here? `transformers` doesn't seem to be uploading to our channel anymore, and we have our package on conda-forge. That recipe is more up-to-date.

Reply:
Right now the build works well and is automatic; I'd like to draft a small message to share with the rest of the team regarding conda support and whether we want to continue to support it through our own channel or not. In the meantime, I'd like to keep it here.